lib/fablicop.rb

```diff
@@ -1,8 +1,18 @@
 # frozen_string_literal: true
 
-require 'fablicop/cli'
-require 'fablicop/version'
+require 'rubocop'
+require_relative "fablicop/cli"
+require_relative 'fablicop/inject'
 
 module Fablicop
-  # Your code goes here...
+  class Error < StandardError; end
+  PROJECT_ROOT   = Pathname.new(__dir__).parent.expand_path.freeze
+  CONFIG_DEFAULT = PROJECT_ROOT.join('.rubocop.yml').freeze
```

Comment:
`.rubocop.yml` is intended for linting our `fablicop` itself, so I think we should put `config/default.yml` to provide default configurations. Or, we should just put the cop into `config/.base_rubocop.yml` if we don't consider the case users want to use `fablicop` without `Fablicop/UnexpectedIdentifiers`.